haros/haros.py

```diff
@@ -739,9 +748,9 @@ def _save_database(self):
 ###############################################################################
 
 class HarosVizRunner(HarosRunner):
-    def __init__(self, haros_dir, server_dir, host_str, headless, log = None,
+    def __init__(self, haros_dir, config_dir, server_dir, host_str, headless, log = None,
```

Comment:
Same as above. Replace `config_dir` with `config_path`, since you are expecting a full path to the config file.

Also, the call to this runner in `HarosLauncher.command_viz` was not changed to reflect the additional argument.

Reply:
Thanks for the notice for the `HarosLauncher.command_viz` - I changed the commit accordingly.